nmostesting/suites/IS1301Test.py

```diff
@@ -0,0 +1,316 @@
+        if not url:
+            msg = f"Can't get annotation url for {resource}"
+            self.log(f"    FAIL {msg}")
+            return test.FAIL(msg)
+
+        node_url = self.get_url(self.node_url, resource)
+        if not url:
+            msg = f"Can't get node url for {resource}"
+            self.log(f"    FAIL {msg}")
+            return test.FAIL(msg)
+
+        valid, r = self.get_resource(url)
+        msg = f"SAVE initial: {r}"
+        self.log(f"    {msg}")
+        if valid:
+            prev = r
+            initial = copy.copy(r)
+            initial.pop('id')
+            initial.pop('version')
+            initial['tags'] = strip_tags(initial['tags'])
+        else:
+            self.log("    FAIL")
+            return test.FAIL(f"Can't {msg}")
+
+        valid, r = self.set_resource(url, node_url, {object: None}, prev)
+        msg = f"RESET to default and save: {r}"
+        self.log(f"    {msg}")
+        if valid:
+            default = prev = r
+        else:
+            self.log("    FAIL")
+            return test.FAIL(f"Can't {msg}")
+
+        value = TAGS_MAX_VALUE if object == "tags" else STRING_MAX_VALUE
+        valid, r = self.set_resource(url, node_url, {object: value}, prev)
+        msg = f"SET MAX value and expected complete response: {r}"
+        self.log(f"    {msg}")
+        if valid:
+            prev = r
+        else:
+            self.log("    FAIL")
+            return test.FAIL(f"Can't {msg}")
+
+        value = TAGS_OVER_MAX_VALUE if object == "tags" else STRING_OVER_MAX_VALUE
+        valid, r = self.set_resource(url, node_url, {object: value}, prev)
+        msg = f"SET >MAX value and expect truncated response: {r}"
+        self.log(f"    {msg}")
+        if valid:
+            prev = r
+            self.log(f"    {msg}")
+            if object == "tags" and not TestHelper.compare_json(r[object], TAGS_MAX_VALUE) or r[object] != STRING_MAX_VALUE:
+                return test.FAIL(f"Can't {msg}")
+        else:
+            self.log("    FAIL")
+            return test.FAIL(f"Can't {msg}")
+
+        valid, r = self.set_resource(url, node_url, {object: None}, prev)
+        msg = f"RESET again and compare: {r}"
+        self.log(f"    {msg}")
+        if valid:
+            if object == "tags" and not TestHelper.compare_json(default[object], r[object]) or default[object] != r[object]:
+                self.log("    FAIL")
+                return test.FAIL("Second reset gives a different default value.")
+            prev = r
+        else:
+            self.log("    FAIL")
+            return test.FAIL(f"Can't {msg}")
+
+        # restore initial for courtesy
+        self.set_resource(url, node_url, initial, prev)
+
+        self.log("    PASS")
+        return test.PASS()
+
+    def test_01_01(self, test):
+        """ Annotation test: self/label (reset to default, set 64-byte value, set >64-byte, check IS04+version)"""

```

Comment:
These strings appear in the UI to describe the test, so let's wordsmith somewhat, @jonathan-r-thorpe?